### PR TITLE
enh: allow uses_qform in FLIRT, ApplyXFM interfaces

### PR DESCRIPTION
**Pull request by mgxd**
Addresses #1317 - allows use of `apply_xfm` with either `in_matrix_file` or `uses_qform` args

### CODE REVIEW

File: nipype/interfaces/fsl/preprocess.py [hunk 1]
```diff
@@ -561,13 +562,18 @@ def _parse_inputs(self, skip=None):
         if isdefined(self.inputs.save_log) and self.inputs.save_log:
             if not isdefined(self.inputs.verbose) or self.inputs.verbose == 0:
                 self.inputs.verbose = 1
+        if isdefined(self.inputs.apply_xfm) and self.inputs.apply_xfm:
+            if not self.inputs.in_matrix_file and not self.inputs.uses_qform:
```
Comment (satra) [inline nipype/interfaces/fsl/preprocess.py]: shouldn't this be `if not isdefined(self.inputs.in_matrix_file) and not (isdefined(self.inputs.uses_qform)  and not self.inputs.uses_qform)`

Comment (mgxd) [inline nipype/interfaces/fsl/preprocess.py]: is `isdefined` necessary in this case? I added a test for both cases of using apply_xfm

Comment (satra) [inline nipype/interfaces/fsl/preprocess.py]: what if someone sets them to `Undefined`

```
In [1]: from traits.api import Undefined

In [2]: not Undefined
Out[2]: False
```

Comment (satra) [inline nipype/interfaces/fsl/preprocess.py]: however the check with not appears to work now

Comment (satra) [inline nipype/interfaces/fsl/preprocess.py]:
```
In [20]: foo.inputs.wmnorms = Undefined

In [21]: not foo.inputs.wmnorms
Out[21]: True
```
haven't figured out why yet

Comment (mgxd) [inline nipype/interfaces/fsl/preprocess.py]: @satra should this be changed?

**Pull request merged**